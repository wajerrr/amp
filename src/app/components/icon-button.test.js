import React from 'react';
import renderer from 'react-test-renderer';
import IconButton from './icon-button';

describe('IconButton', () => {
  it('should match snapshot', () => {
    const tree = renderer
      .create(<IconButton onProps="toggleMenu" icon="hambuger" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
