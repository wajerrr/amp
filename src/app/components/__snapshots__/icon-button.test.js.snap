// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconButton should match snapshot 1`] = `
<button
  className="css-7g0he1"
  on="toggleMenu"
  tabIndex="0"
/>
`;
